### PR TITLE
Use UPSTREAM_CLUSTER_RAW for access log format

### PR DESCRIPTION
**Pull request by keithmattix**
**Please provide a description of this PR:**
This should be the last in the series of PRs around stats and delimiters. This PR utilizes the new UPSTREAM_CLUSTER_RAW formatter that I added in Envoy to ignore the alt stat name for access logs (but not metrics). This keeps access logs consistent for users, and they don't have to see a random semicolon show up.

### CODE REVIEW

File: pilot/pkg/networking/core/accesslog.go [hunk 1]
```diff
@@ -236,13 +236,13 @@ func buildAccessLogFilter(f ...*accesslog.AccessLogFilter) *accesslog.AccessLogF
 	}
 }
 
-func (b *AccessLogBuilder) buildListenerFileAccessLog(mesh *meshconfig.MeshConfig) *accesslog.AccessLog {
+func (b *AccessLogBuilder) buildListenerFileAccessLog(mesh *meshconfig.MeshConfig, proxyVersion *model.IstioVersion) *accesslog.AccessLog {
 	if cal := b.cachedListenerFileAccessLog(); cal != nil {
```
Comment (howardjohn) [inline pilot/pkg/networking/core/accesslog.go:240]: do we need to handle the caching aspect here? maybe only use/set the cache if we are on GE123 so we don't need to handle two caches

**Review by howardjohn — COMMENTED** [review]
yay

File: pilot/pkg/networking/core/accesslog.go (hunk 1 above)
Comment (keithmattix) [reply pilot/pkg/networking/core/accesslog.go:240]: Ahh, it's a shared access log builder! Good catch; agreed - we should only set the cache on one version

**Review by howardjohn — APPROVED** [review]

**Pull request merged**